api_testrail.py

```diff
@@ -86,7 +86,29 @@ def test_runs(self, testrail_project_id, start_date='', end_date=''):
         if end_date:
             before = dt.convert_datetime_to_epoch(end_date)
             date_range += '&created_before={0}'.format(before)
-        return self.client.send_get('get_runs/{0}{1}'.format(testrail_project_id, date_range)) # noqa
+        return self.client.send_get('get_runs/{0}{1}'.format(testrail_project_id, date_range))  # noqa
+
+    def get_plans(self, testrail_project_id, start_date='', end_date=''):
```

Comment:
Can you add a section here `# API: Plans`, similar to the previous ones. Also, rename the method for consistency: `test_plans()`